libmscore/tempotext.h

```diff
@@ -46,11 +47,13 @@ class TempoText : public Text  {
 
       qreal tempo() const        { return _tempo;      }
       void setTempo(qreal v);
+      bool isRelative() { return _relative!=0; }
```

Comment:
If we change it to 1.0, how close does a float 1.0 get to an integer 1? Does a 1 have an exact representation of float? I'm not sure and it may even depend on the platform, so we'd better use a delta here